docs/dev/framework/content-elements.md

```diff
@@ -143,6 +143,47 @@ $GLOBALS['TL_LANG']['CTE']['my_content_element'] = [
 ```
 
 
+## Annotation
+
+{{< version "4.8" >}}
+
+Instead of tagging the content element controller service via the service configuration,
+the service tag can also be configured through annotations, as already used in the 
+code example above. The annotation can be used on the class of the content element 
```

Comment:
> can be used on the class

if that class is invokable (has `__invoke` method), or basically if it extends the abstract controller